src/ert/shared/share/ert/forward-models/res/script/ecl_config.py

```diff
@@ -203,8 +206,38 @@ class FlowConfig(EclConfig):
 
     def __init__(self):
         config_file = os.getenv("FLOW_SITE_CONFIG", default=self.DEFAULT_CONFIG_FILE)
+        if not Path(config_file).exists():
+            config_file = self.init_flow_config()
         super().__init__(config_file, simulator_name="flow")
 
+    @staticmethod
+    def locate_flow_binary() -> str:
+        """Locate the path for a flow executable.
+
+        Returns the empty string if there is nothing to be found in $PATH."""
+        rhel_version = "7"
+        if "el8" in platform.release():
+            rhel_version = "8"
+        candidates = ["flow", f"/project/res/x86_64_RH_{rhel_version}/bin/flowdaily"]
```

Comment:
I don't think we need to expose `/project/res/` details to the rest of the world. I think assuming that you have `flow` on `PATH` should be sufficient for most use cases :slightly_smiling_face: